tests/test_mathematical.py

```diff
@@ -402,6 +402,35 @@ def test_negative(data, dtype):
     assert_allclose(result, expected)
 
 
+def test_negative_boolean():
+    dpnp_a = dpnp.array([True, False])
+
+    with pytest.raises(TypeError):
+        dpnp.negative(dpnp_a)
+
+
+@pytest.mark.parametrize(
+    "data",
+    [[2, 0, -2], [1.1, -1.1]],
+    ids=["[2, 0, -2]", "[1.1, -1.1]"],
+)
+@pytest.mark.parametrize("dtype", get_all_dtypes(no_bool=True, no_complex=True))
```

Comment:
I think we still need to secure that fallback works with complex type
```suggestion
@pytest.mark.parametrize("dtype", get_all_dtypes(no_bool=True))
@pytest.mark.usefixtures("allow_fall_back_on_numpy")
```